front/src/app/contents/forms/convention/formConvention.tsx

```diff
@@ -306,6 +306,12 @@ const beneficiarySection = (internshipKind: InternshipKind) => ({
     id: beneficiarySectionIds.emergencyContactEmail,
     placeholder: "Ex : contact@urgence.com (optionnel)",
   },
+  "signatories.beneficiary.address": {
+    label: "Adresse du candidat",
+    id: beneficiarySectionIds.address,
+    placeholder: "Ex : 10 Rue de la Paix, 75001 Paris",
+    required: true,
+  },
```

Comment:
Ce champ est seulement obligatoire dans le cas où le bénéficiaire habite en Auvergne Rhône Alpe.
Donc je propose de mettre ici:

```suggestion
  "signatories.beneficiary.address": {
    label: "Adresse du candidat",
    hintText: "Si vous résidez en Auvergne Rhône Alpe, il est obligatoire de saisir votre adresse."
    id: beneficiarySectionIds.address,
    placeholder: "Ex : 10 Rue de la Paix, 75001 Paris",
  },
```

Reply:
Finalement le champ est obligatoire pour tous les mini-stages (voir message discord 24/09 canal dev) et il n'y a plus de restriction sur la région